lib/vm/stack.js

```diff
@@ -80,4 +81,18 @@ module.exports = class Stack {
     const i = this._store.length - position
     this.push(this._store[i])
   }
+
+  _isValidValue (value) {
+    if (BN.isBN(value)) {
+      if (value.lte(ethUtil.MAX_INTEGER)) {
+        return true
+      }
+    } else if (Buffer.isBuffer(value)) {
+      if (value.length <= 32) {
+        return true
+      }
+    }
+
+    return false
+  }
```

Comment:
Should we maybe directly put stuff like this in ``vm/util.js`` or similar?

Reply:
Maybe when we start using it in other places?

Reply:
Yeah, also completely makes sense.